### PR TITLE
doc: add security.md file

### PR DESCRIPTION
**Pull request by Rahul-4480**

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Security
+
+GitHub takes the security of our software products and services seriously, including all of the open source code repositories managed through our GitHub organizations, such as [GitHub](https://github.com/GitHub).
```
Comment (ashwinimanoj) [inline SECURITY.md:3]: Replace references to Github with Infraspec

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+Even though [open source repositories are outside of the scope of our bug bounty program](https://bounty.github.com/index.html#scope) and therefore not eligible for bounty rewards, we will ensure that your finding gets passed along to the appropriate maintainers for remediation.
+
+## Reporting Security Issues
+
+If you believe you have found a security vulnerability in any GitHub-owned repository, please report it to us through coordinated disclosure.
+
+**Please do not report security vulnerabilities through public GitHub issues, discussions, or pull requests.**
+
+Instead, please send an email to <rohit@infraspec.dev>.
```
Comment (ashwinimanoj) [inline SECURITY.md:13]: could we use `security@infrapsec.dev` instead?

File: SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# Security
+
+GitHub takes the security of our software products and services seriously, including all of the open source code repositories managed through our GitHub organizations, such as [GitHub](https://github.com/GitHub).
+
+Even though [open source repositories are outside of the scope of our bug bounty program](https://bounty.github.com/index.html#scope) and therefore not eligible for bounty rewards, we will ensure that your finding gets passed along to the appropriate maintainers for remediation.
```
Comment (ashwinimanoj) [inline SECURITY.md:5]: we can just mention that the finding will be passed appropriately, don't need to mention bug bounty

File: SECURITY.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+* Any special configuration required to reproduce the issue
+* Step-by-step instructions to reproduce the issue
+* Proof-of-concept or exploit code (if possible)
+* Impact of the issue, including how an attacker might exploit the issue
+
+This information will help us triage your report more quickly.
+
+## Policy
+
+See [GitHub's Safe Harbor Policy](https://docs.github.com/en/site-policy/security-policies/github-bug-bounty-program-legal-safe-harbor#1-safe-harbor-terms)
```
Comment (ashwinimanoj) [inline SECURITY.md:29]: we can omit this line

**Review by ashwinimanoj — CHANGES_REQUESTED** [review]

File: SECURITY.md (hunk 1 above)
Comment (Rahul-4480) [reply SECURITY.md]: Done 63f15249abf6fbbd914c2d5932e619df45e1c715

File: SECURITY.md (hunk 2 above)
Comment (Rahul-4480) [reply SECURITY.md]: Done 9a5bf19eb2ccf7509c1c444fe88f49a32d47f332

File: SECURITY.md (hunk 3 above)
Comment (Rahul-4480) [reply SECURITY.md]: Done e3c6164e73d90cbdd6e41d3732e0ed8720b4c164

File: SECURITY.md (hunk 4 above)
Comment (Rahul-4480) [reply SECURITY.md]: Done fbb38bc3cef31a626948b64ded11cba3d7912aa6

File: SECURITY.md (hunk 1 above)
Comment (ashwinimanoj) [reply SECURITY.md]: I still see Github mentioned, you can use a find replace on all references to Github. 
"GitHub takes the security of.." should be "Infraspec takes the security of.." as this doc is saying we (Infraspec) take security seriously.
for reference, [here](https://github.com/microsoft/.github/blob/main/SECURITY.md)

File: SECURITY.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
+# Security
+
+GitHub takes the security of our software products and services seriously, including all open source code repositories managed through our GitHub organizations, such as [GitHub](https://github.com/infraspecdev/).
```
Comment (vjdhama) [inline SECURITY.md:3]: Change to:

Infraspec takes the security of our software products and services seriously, including all open-source code repositories managed through our GitHub organization, [Infraspec](https://github.com/infraspecdev/).

File: SECURITY.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+If you believe you have found a security vulnerability in any GitHub-owned repository, please report it to us through coordinated disclosure.
+
+**Please do not report security vulnerabilities through public GitHub issues, discussions, or pull requests.**
+
+Instead, please send an email to <security@infrapsec.dev>.
+
+Please include as much of the information listed below as you can to help us better understand and resolve the issue:
+
+* The type of issue (e.g., improper input handling, token exposure, unauthorized access to GitHub API, or API abuse in the spell and grammar checking process)
```
Comment (vjdhama) [inline SECURITY.md:17]: Change examples to remove GitHub examples like unauthorized access to GitHub API.

**Review by vjdhama — CHANGES_REQUESTED** [review]

File: SECURITY.md (hunk 5 above)
Comment (Rahul-4480) [reply SECURITY.md]: Done d26b09f780dedbbfbe4dd7084e4186c29929ad4a

File: SECURITY.md (hunk 6 above)
Comment (Rahul-4480) [reply SECURITY.md]: Done b473732540c72f0015f987b2ab84b7fc401a831c

File: SECURITY.md (hunk 1 above)
Comment (Rahul-4480) [reply SECURITY.md]: Done 8431e0c1c455b8e0c00535952e711fa460562476

**Review by ashwinimanoj — APPROVED** [review]

**Pull request merged**